### PR TITLE
reorder steps in the Dockerfile to reduce the image size 

### PR DESCRIPTION
**Pull request by balda-rdx**
This reorder steps in the Dockerfile to reduce the image size. By unziping in the final stage we are adding an extra layer to the image that's not needed. it is better to unzip the artifacts in the build stage and then just copy, that way we prevent the extra layer

Also the user and groups creation were removed from the config script and moved to the Dockerfile. If there are any errors on those 2, we should be able to get them at build time and not at run time like now

### CODE REVIEW

File: docker/build_scripts/config_radixdlt.sh [hunk 1]
```diff
@@ -1,20 +1,6 @@
 #!/bin/bash
-
 set -ex
 
-# Need this to run on Alpine (grsec) kernels without crashing
-find /usr -type f -name java -exec setfattr -n user.pax.flags -v em {} \;
-
-# Sets USER_ID to LOCAL_USER_ID if provided, else set it to 999
```
Comment (shambupujar) [inline docker/build_scripts/config_radixdlt.sh:8]: The lines below from down here should be kept intact. Tehy allow us to run the container as radixdlt user

Comment (shambupujar) [reply docker/build_scripts/config_radixdlt.sh:8]: Those steps needed at runtime as the vars are only avaliable on run time and they vary on individual node runners.

Comment (balda-rdx) [reply docker/build_scripts/config_radixdlt.sh:8]: @shambupujar so I only wanted to move the user creation to the Dockerfile, would you be ok with keeping the user and group creation there?
As of env vars, would you please let me know which ones are needed? I've tested the docker image created and it is starting just fine

Comment (balda-rdx) [reply docker/build_scripts/config_radixdlt.sh:8]: resolving as per the agreement on slack

**Pull request merged**